api/service/syncing/syncing.go

```diff
@@ -580,7 +596,7 @@ func (ss *StateSync) generateNewState(bc *core.BlockChain, worker *worker.Worker
 		if block == nil {
 			break
 		}
-		err = ss.UpdateBlockAndStatus(block, bc, worker)
+		err = ss.UpdateBlockAndStatus(block, bc, worker, false)
```

Comment:
if default is false, what if a node sent a lot of invalid block to cause new node roll back?

Reply:
false is the same logic as before. This change is to make beacon sync work.

The state sync'ing will need more careful work in general.

Reply:
I see